sort/src/main.rs

```diff
@@ -10,24 +10,17 @@ mod cli;
 fn main() {
     let matches = cli::create_app().get_matches();
 
-    let result = main_sort(matches);
-
-    let exit_code = match result {
-        Ok(_) => 0,
-        Err(err) => {
-            eprintln!("sort: {:?}", err);
-            1
-        },
-    };
-    std::process::exit(exit_code);
+    main_sort(matches).unwrap_or_else(|err| {
+        eprintln!("sort: {:?}.", err);
+        std::process::exit(2);
```

Comment:
Return code for errors in `sort` is `2` instead of `1`, because `1` is used for `-c` response.